spec/sinatra_ar_crud_lab_spec.rb

```diff
@@ -0,0 +1,102 @@
+require 'spec_helper'
+
+describe "Blog Post App" do
+  let(:post_name) { "Hello World" }
+  let(:post_content) { "This is my first blog post!!!" }
+
+  before do
+    @post1 = Post.create(:name => post_name, :content => post_content)
+    @post2 = Post.create(:name => "second post", :content => "i'm a really good blogger")
+  end
+
+  describe "Create Action" do 
+
+    it "creates a new blog post" do
+      visit '/posts/new'
+
+      fill_in :name, :with => "my favorite blog post"
+      fill_in :content, :with => "blogging!!!!"
+
+      click_button 'submit'
+
+      expect(Post.all.count).to eq(3)
+      expect(Post.last.name).to eq("my favorite blog post")
+    end
+
+  end
+
+  describe "Read Action " do
+
+    it 'index action responds with a 200 status code' do
+      get "/posts"
+      expect(last_response.status).to eq(200)
+    end
+
+    it "index action displays all the blog posts" do
```

Comment:
These `it` blocks should read like a sentence. "it index action displas all the blog posts" doesn't quite work. You have two options. Either you do another nested `describe` block. One for "index action" and one for "show action" and then an `it` like "should display all the blog posts" under the index action describe. Or! just do something like `it` "should display all blog posts when requesting the index"